www.foia.gov/_sass/modules/_typeahead.scss

```diff
@@ -0,0 +1,40 @@
+.twitter-typeahead {
+  width: 100%;
+
+  .tt-menu {
+    background-color: #fff;
+    border: 0.0625rem solid #ccc;
```

Comment:
Are these out-of-the-box styles? Our base `rem` is probably different from what this is expecting, if so. Also, this should probably be `$color-gray-light`.

Reply:
The typeahead styles are based on twitter bootstrap, so they completely fall apart with the WDS. I think I copy/pasted these from the discovery prototype since they seemed to work reasonably. I'm thinking these styles will get another pass when we do the landing page.

Reply:
... I fixed the color, but can you explain the rem thing to me?